MovingBlocks/Terasology
# Added a new Biomes Metric mode and fixed OnBiomeChangedEvent

### Contains

Added a new Biomes Metric mode to the debug mode showing information regarding the current biome in which the user is present. 
Also fixes the OnBiomeChanged event and block extra data storage by passing the chunk of the block as an argument to the setExtraData function. 
### How to test

Load/Create a world, Press F3 to toggle the debug mode, Press F4 to cycle between the metrics modes till the Biomes Metrics Mode is reached. This should contain the current Biome. Test by moving around with this enabled and watch the biome name change.
